.github/workflows/release.yml

```diff
@@ -74,9 +63,10 @@ jobs:
                   echo -n "${{ secrets.OSSRH_GPG_SECRET_KEY }}" | base64 --decode > $GITHUB_WORKSPACE/release.gpg
             - name: Publish to Maven Central
               run: |
-                  ./gradlew publishToSonatype -Psemver.overrideVersion=${{ steps.ci-release-create-outputs.outputs.semVersion }} $(if [ "${{github.event.release.prerelease}}" = "true" ]; then echo 'closeSonatypeStagingRepository'; else echo 'closeAndReleaseSonatypeStagingRepository'; fi) \
+                  ./gradlew publishToSonatype -Pversion="$VERSION" $(if [ "${{github.event.release.prerelease}}" = "true" ]; then echo 'closeSonatypeStagingRepository'; else echo 'closeAndReleaseSonatypeStagingRepository'; fi) \
```

Comment:
I have yet to test this exact logic, but I _think_ it should work 🤞🏽

Reply:
🤞